process_scraped_data/run_process_target_year.sh

```diff
@@ -43,8 +43,8 @@ if [ ! -d ${CORENLP_OUTPUT}  ]; then
 fi
 
 ## process the results, we do this always since this code is likely to change
-QUOTE_RES_FILE="${DIR}/../data/scraped_data/quote_table_raw_${TARGET_YEAR}.tsv"
-RScript ${DIR}/process_corenlp_quotes_corenlp_output.R ${CORENLP_OUTPUT} ${QUOTE_RES_FILE}
+#QUOTE_RES_FILE="${DIR}/../data/scraped_data/quote_table_raw_${TARGET_YEAR}.tsv"
+#RScript ${DIR}/process_corenlp_quotes_corenlp_output.R ${CORENLP_OUTPUT} ${QUOTE_RES_FILE}
```

Comment:
Sometimes it's useful to leave code commented out for some reason. Just checking if this is the case here, or if you can remove it.

Reply:
ah, this should stay. it was commented out here because I was testing something.